services/mgmt/vmware/examples/private_cloud_list.rs

```diff
@@ -10,25 +10,76 @@ https://docs.microsoft.com/rest/api/vmware/privateclouds/list
 cargo run --package azure_mgmt_vmware --example private_cloud_list
 */
 
+use azure_core::http::Url;
 use azure_identity::AzureCliCredential;
-use futures::stream::StreamExt;
-use std::sync::Arc;
+use futures::TryStreamExt;
 
 #[tokio::main]
 async fn main() -> Result<(), Box<dyn std::error::Error>> {
-    let subscription_id = AzureCliCredential::get_subscription().await?;
-    let credential = Arc::new(AzureCliCredential::new());
+    let subscription_id = &std::env::var("AZURE_SUBSCRIPTION_ID").expect("AZURE_SUBSCRIPTION_ID required");
+    let credential = AzureCliCredential::new(None)?;
     let client = azure_mgmt_vmware::Client::builder(credential).build()?;
 
     let mut count = 0;
-    let mut clouds = client.private_clouds_client().list_in_subscription(subscription_id).into_stream();
-    while let Some(clouds) = clouds.next().await {
-        let clouds = clouds?;
-        count += clouds.value.len();
-        for cloud in clouds.value {
-            println!("{:?}", cloud.tracked_resource.resource.id);
+    let clouds_client = client.private_clouds_client();
+    let mut cloud_pager = clouds_client.list_in_subscription(subscription_id).pager()?;
```

Comment:
This can probably just be `clouds`. This example can go back to just listing clouds. I was trying out other APIs.